Placeeo/Controller/MainViewController.swift

```diff
@@ -6,22 +6,30 @@
 //
 
 import UIKit
+import CoreLocation
+import MapKit
 
-class ViewController: UIViewController {
+class MainViewController: MapViewController{
```

Comment:
why did you change `UIViewController` to `MapViewController`?

Reply:
i think than in future i need to inherit from MapVC

Reply:
not really. Main and Map view controllers must not to depend on each other, thus none of them should inherit each other. let's create an instance of MapVC inside MainVC and call it's methods to show places

Reply:
done!